design/ceph/object/zone.md

```diff
@@ -128,6 +130,10 @@ The following variables can be configured in the ceph-object-zone resource.
 
 - `zoneGroup`: The zone group named in the `zoneGroup` section of the ceph-realm resource the zone is a part of.
 
+- `customEndpoints`:  Specify the endpoint(s) that will accept multisite replication traffic for this zone. You must also include the port in the definition. For example: "https://my-object-store.my-domain.net:443". By default, Rook will set this to the DNS name of the ClusterIP Service created for the CephObjectStore that corresponds to this zone.
```

Comment:
Since the full description is in the documention I don't think we need to duplicate it to this design doc. Maybe a 1-line description is fine for the design doc.